blogposts/2024-07-18-running-rnw-from-a-mac.md

```diff
@@ -0,0 +1,86 @@
+1. You will also need a work or school MSA (Microsoft Account). Personal or guest MSA’s cannot be used with the Dev Box portal at this time.
+1. Your machine will need to be managed with [Microsoft Intune](https://www.microsoft.com/en-us/security/business/Microsoft-Intune).
+
+I followed Microsoft's documentation to create a dev box using Microsoft Dev Box. I found the documentation to be straightforward to follow; there are also several tutorial videos online from developers, if you prefer to see someone walk through the steps live. You'll need to complete two steps of documentation to get your dev box up and running: [Setting Up Your Dev Box Service](https://learn.microsoft.com/en-us/azure/dev-box/quickstart-configure-dev-box-service) and [Creating a Dev Box](https://learn.microsoft.com/en-us/azure/dev-box/quickstart-create-dev-box). These steps should take about an hour to complete.
+
+Once your dev box is running, you will be able to use the virtual machine as if you are on a Windows computer.
+
+From here, you can follow the ["System Requirements"](https://microsoft.github.io/react-native-windows/docs/rnw-dependencies) and ["Getting Started"](https://microsoft.github.io/react-native-windows/docs/getting-started) documentation for React Native for Windows. Dev box will not have access to your local files, so you will need to redownload any source code that you need to get started developing. When you are finished with your development session and have shut down the virtual machine in the Dev Box portal, your machine's state will be saved for next time.
+
+![](assets/2024-07-18-running-rnw-from-a-mac/Screenshot-MicrosoftDevBox.jpg)
+
+## Takeaways about Microsoft Dev Box
+
+Microsoft Dev Box is a great tool when your team is ready for a larger level of investment in the Windows platform. Its feature offerings of cloud-based infrastructure, pre-configured dev boxes, and Intune secured workstations are great for enterprise-scale development.
+
+Microsoft Dev Box does require several admin and configuration steps to get started, so if a developer is looking to quickly experiment with RNW within a VM, Parallels or another virtual machine offering may be a good option to get started with. Then, once the developer is ready to scale their Windows VM usage across many developers to build large production experiences, Microsoft Dev Box will become a better option.
+
+## Overall Takeaways
+
+In this post, I looked at two options for building React Native for Windows apps from Mac devices using Windows VMs. As I close out my content for today, I want to emphasize a few key points. Below is a table which captures the scenario comparisons between Microsoft Dev Box and Parallels.
+
+| Feature                                          | Microsoft Dev Box | Parallels |
+| ------------------------------------------------ | ----------------- | --------- |
+| Locally installed virtual machine software       |                   | YES       |
```

Comment:
Thoughts on replacing these with emoji checkmarks?

Reply:
Can definitely swap to that, how do you add an emoji to the markdown? Do you have to save the asset image?

Reply:
Is it possible to just use the emoji from emoji picker? ✅